pypga/modules/migen/spimachine.py

```diff
@@ -158,15 +176,16 @@ def __init__(self, data_width=32, div_width=8):
                 self.cs_next.eq(1),
                 cg.count.eq(1),
                 cg.extend.eq(1),
-                self.clk_next.eq(1),
+                self.clk_next.eq(0),
```

Comment:
IMHO the clock polarity is wrong according to how it's usually defined.

Reply:
can be fixed here or in https://github.com/pypga/pypga/blob/e40dded1e8dbbe9def18d4e59f8c2bf2108616a2/pypga/modules/migen/spi.py#L60 by inverting the clock-polarity value.